nekoyume/Assets/_Scripts/Blockchain/BlockManager.cs

```diff
@@ -84,32 +84,52 @@ public static Block ImportBlock(string path)
 #endif
         }
 
-        public static async Task<Block> ImportBlockAsync(string path)
+        public static async UniTask<Block> ImportBlockAsync(string path)
         {
-            // If it contains "main", it is main-net genesis block. client have to contain main-net genesis block.
-            var localPath = path.Contains("main")
-                ? Platform.GetStreamingAssetsPath("genesis-block-9c-main")
-                : path;
+            var uri = new Uri(path);
+            // NOTE: Check cached genesis block in local path.
+            string localPath;
+            if (uri.IsFile)
+            {
+                localPath = path;
+            }
+            else if (uri.AbsolutePath.Equals("/genesis-block-9c-main") ||
```

Comment:
Could we apply generalized `planets/<id>` convention against `genesis-block-9c-main` too?

Reply:
The [planet registry](https://planets.nine-chronicles.com/planets/index.json) still returns "https://release.nine-chronicles.com/genesis-block-9c-main".
And it also covered "https://planets.nine-chronicles.com/planets/0x000000000000/genesis".